packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1034,7 +1034,9 @@ class CupertinoDialogAction extends StatelessWidget {
     this.isDestructiveAction = false,
     this.textStyle,
     @required this.child,
-  }) : assert(child != null);
+  }) : assert(child != null),
+       assert(isDefaultAction != null),
+       assert(isDestructiveAction != null);
```

Comment:
On lines 1049 and 1054, add more lines describing their defaulting behavior:

`Defaults to false and cannot be null`